website/addons/s3/views/crud.py

```diff
@@ -14,6 +16,7 @@
 @must_have_permission('write')
 def create_bucket(auth, node_addon, **kwargs):
     bucket_name = request.json.get('bucket_name', '')
+    bucket_location = request.json.get('bucket_location', DEFAULT_BUCKET_LOCATION['value'])
```

Comment:
It looks like you removed the `DEFAULT_BUCKET_LOCATION` setting without removing it from the code.